ARExperiment/ARExperiment/SizeComparisonViewController.swift

```diff
@@ -3,17 +3,61 @@ import SceneKit
 import ARKit
 
 class SizeComparisonViewController: UIViewController, ARExperimentSessionHandler {
+
+    enum CubeModel {
+        case big    //5x5
+        case medium //1x1
+        case small  //0.1x0.1
+        
+        var filename: String {
+            switch self {
+            case .big:
+                return "cube_5"
+            case .medium:
+                return "cube_1"
+            case .small:
+                return "cube_0.1"
+            }
+        }
+        
+        var textFilename: String {
+            switch self {
+            case .big:
+                return "text_5"
+            case .medium:
+                return "text_1"
+            case .small:
+                return "text_0.1"
+            }
+        }
+    }
     
+    enum Scale {
+        case arWorldScale
+        case realWorldScale
+        
+        var persentage: Float {
+            switch self {
+            case .arWorldScale:
+                return 1
+            case .realWorldScale:
+                return 0.01
+            }
+        }
+    }
+
     @IBOutlet var sceneView: ARSCNView!
-    @IBOutlet var segmentControl: UISegmentedControl!
+    @IBOutlet weak var realWorldMeasuresSegmentedControl: UISegmentedControl!
+    @IBOutlet weak var ARMeasuresSegmentedControl: UISegmentedControl!
+
     private var scene: SCNScene?
     private let arSessionDelegate = ARExperimentSession()
     private let arModel = ARViewModel()
     private let fileName = "measuring-units"
     private let fileExtension = "scn"
-    private var currentCubeName = ""
-    private var currentTextName = ""
-    
+    private var currentCube = CubeModel.big //Default model
```

Comment:
make sure that this is the default for the segment too 👍

Reply:
yeah